main.go

```diff
@@ -135,7 +136,7 @@ func main() {
 
 			log.Printf("Invoking (%s) on topic: %q, value: %q", gatewayURL, topic, data)
 
-			controller.Invoke(topic, &data)
+			controller.Invoke(topic, &data, http.Header{})
```

Comment:
@alexellis Tbh I'm not really sure about this, I do not fully understand how and why we need to pass headers there, this call here is fine or should I change something?

Reply:
In another PR we could consider what X-Key=Value kind of headers we could pass from the MQTT message like the originating server name for instance? Or anything else that might be useful as a recipient of a message from this connector